src/SDAO.sol

```diff
@@ -157,6 +164,25 @@ contract SDAO {
         emit Deny(usr);
     }
 
+    /**
+     * @notice Updates token parameters.
+     * @dev There are no mechanisms to prevent governance from changing token parameters more than once.
+     *  We assume that the enforcement will be handle off-chain through governance artifacts.
```

Comment:
Can we at least consider adding a forfeit function that blocks governance ability to change the name and symbol?
This changing ability is unconventional, and I can imagine a point in time when this becomes contreversial (whether it's integrations that deem it unsafe as it might allow to disguise as another token, or just bad PR).

BTW if we do add that then it might make more sense to drop the `file` pattern and instead have one function to change both `name` and `symbol` (while checking that the right for that was not forfeited).

Reply:
Someone had proposed we Kelvin-version it, but the idea was dismissed in favor of simplicity. At least that was my understanding.